draft-demarco-oauth-status-attestations.md

```diff
@@ -369,8 +372,25 @@ MUST demonstrate the proof of possession of the Digital Credential to
 the Credential Issuer.
 
 The proof of possession is made by enclosing the Digital Credential in an
-object (JWT) signed with the private key that its related public key is
-referenced in the Digital Credential.
+object (JWT or CWT) signed with the private key that its related public key is referenced in the Digital Credential.
+
+When the JWT or CWT format are used, the JWT/CWT MUST contain the parameters defined in the following table.
+
+| Header | Description | Reference |
+| --- | --- | --- |
+| **typ** | It MUST be set to `status-attestation+jwt` when JWT format is used. It MUST be set to `status-attestation+cwt` when CWT format is used. | {{RFC7516}} Section 4.1.1 |
+| **alg** | A digital signature algorithm identifier such as per IANA "JSON Web Signature and Encryption Algorithms" registry. It MUST NOT be set to `none` or any symmetric algorithm (MAC) identifier. | {{RFC7516}} Section 4.1.1 |
+| **kid** | Unique identifier of the JWK or CWT used for the signature of the Status Attestation Request, it MUST match the one contained in the Credential. | {{RFC7515}} |
+
+| Payload | Description | Reference |
+| --- | --- | --- |
+| **iss** | Wallet identifier. | {{RFC9126}}, {{RFC7519}} |
```

Comment:
```suggestion
| **iss** | Wallet identifier. | {{RFC9126}}, {{RFC7519}} |
```

concerns here, since a wallet identifier can be a vector of traceability. We need further explaination about the evidence that this identifier is not unique and absolute, preventing the traceability of the wallet.